README.md

```diff
@@ -42,7 +42,7 @@ Therefore, this application will not run into any issues if it is restarted, res
 
 | Environment variable     | Description | Required | Default |
 |:------------------------ |:----------- |:-------- |:------- |
-| CLUSTER_NAME             | Name of the eks-cluster, used in place of AUTO_SCALING_GROUP_NAMES. Checks for `k8s.io/cluster-autoscaler/<CLUSTER_NAME>: owned` and `k8s.io/cluster-autoscaler/enabled: true` tags on ASG | yes | `""` |
+| AUTODISCOVERY_TAGS       | Comma separated key value string with tags to autodiscover ASGs, used in place of AUTO_SCALING_GROUP_NAMES. | yes | `""` |
```

Comment:
This is a breaking change.

Is there anything you could do to keep supporting `CLUSTER_NAME` so that it remains backward-compatible?
Personally, I would prefer if `CLUSTER_NAME` took priority over `AUTODISCOVERY_TAGS`.

If you look at https://github.com/TwiN/aws-eks-asg-rolling-update-handler/blob/dcc8eb4b57a5444b07b1f550c813526904682dc4/config/config.go#L38-L44 right now, the way that it works is that it:
1. check if `CLUSTER_NAME` is set, if it is, use that
2. else, use `AUTO_SCALING_GROUP_NAMES`

Your change completely breaks the current behavior, and does this instead:
1. check if `AUTODISCOVERY_TAGS` is set, if it is, use that
2. else, use `AUTO_SCALING_GROUP_NAMES`

I would like it if instead, it did the following:
1. check if `CLUSTER_NAME` is set, if it is, it uses that
2. else, checks if `AUTODISCOVERY_TAGS` is set, if it is, use that
3. else, use `AUTO_SCALING_GROUP_NAMES`

Reply:
Latest commit emulates the old `CLUSTER_NAME` using autodiscovery tags.